### PR TITLE
QA8-1289 add corner tests for image

### PR DESCRIPTION
**Pull request by Mykhaylyak**

### CODE REVIEW

File: test/extended/image.js [hunk 1]
```diff
@@ -29,7 +29,33 @@ describe('Image field suit', function () {
         });
 
         it('TC-106 Verify that user can upload a PNG file by clicking the Image upload area', function () {
-            uploadImage(images.PNG);
+            uploadImage(images.default_1);
+            let imageDisplayed = $(sel.imageDisplayed);
+            expect(imageDisplayed).toBeDisplayed();
+        });
+    });
+    describe('Corner cases', function () {
```
Comment (MariiaSosedova) [inline test/extended/image.js:37]: add empty line before line 37

File: data/testData.js [hunk 2]
```diff
@@ -65,7 +65,11 @@ const story = {
 
 const images = {
   "default": '../../data/images/2Mb_main_rect.JPG',
-  "PNG": '../../data/images/1.8MbPNG.png'
+  "default_1": '../../data/images/png-2.png',
```
Comment (MariiaSosedova) [inline data/testData.js:68]: let's rename it just "default1" without "_"?

**Review by MariiaSosedova — CHANGES_REQUESTED** [review]

File: test/extended/image.js (hunk 1 above)
Comment (Mykhaylyak) [reply test/extended/image.js:38]: done

File: data/testData.js (hunk 2 above)
Comment (Mykhaylyak) [reply data/testData.js]: done

**Review by MariiaSosedova — APPROVED** [review]

**Review by andkr5150 — APPROVED** [review]
